packages/node/src/lib/subscribers/subscribers.ts

```diff
@@ -1,20 +1,20 @@
-import type { AxiosResponse } from 'axios';
+import { AxiosResponse } from 'axios';
+import { ButtonTypeEnum, IChannelCredentials } from '@novu/shared';
 import { MarkMessagesAsEnum } from '@novu/shared';
-
-import { WithHttp } from '../novu.interface';
 import {
   IGetSubscriberNotificationFeedParams,
+  IMarkFields,
+  IMarkMessageActionFields,
   ISubscriberPayload,
   ISubscribers,
   IUpdateSubscriberPreferencePayload,
 } from './subscriber.interface';
-
-interface IChannelCredentials {
-  webhookUrl?: string;
-  deviceTokens?: string[];
-}
+import { WithHttp } from '../novu.interface';
 
 export class Subscribers extends WithHttp implements ISubscribers {
+  create(subscriberId: string, data: ISubscriberPayload) {
+    throw new Error('Method not implemented.');
+  }
```

Comment:
Maybe let's remove this for now until we implement it? Like the library users might be super angry to see this error in the runtime. Imagine yourself using the library and seeing the method you need and then you think that you're good and you implement some code, but then get this error during the development. Wtf is the thing that first comes from your mouth? I would be super angry.